### ISSUE TITLE
data and uploader not working

### ISSUE THREAD

**Issue by alexshk** [jonahoffline/filepreviews-ruby#13]
Can't send `data` and `uploader` params with options http://filepreviews.io/docs/endpoints.html

Have `null` in response:
````
...
"user_data": null,
...
````

Sending it with this:

````
config = {options: {
  pages: "1",
  format: "png",
  data: {
    uuid: 'xxxx'
  },
  uploader: {
    destination: 'previews/my_destination'
  }
}}
result = Filepreviews.generate('path-to-file, config)
````

**Comment by jonahoffline** [comment]
@alexshk I'm sorry to reply so late, I hadn't seen this before! 
```Ruby
result = Filepreviews.generate(url, config) # must be an url, not a local path
```

[Additional example](https://github.com/jonahoffline/filepreviews-ruby#options)

**Comment by macfanatic** [comment]
This was an issue on release 2.0.1 and after debugging doesn't appear that this was actually supported, as all arbitrary keys in the `options` hash are eliminated during some of hte processing steps.

I created PR #14 to fix this as easily as possible.

**Comment by jonahoffline** [comment]
@macfanatic thanks for looking into this! 
I merged your PR and released this as version [2.0.2](https://rubygems.org/gems/filepreviews/versions/2.0.2)
